main/com.spotify/spotifyd.js

```diff
@@ -78,6 +78,16 @@ module.exports = class SpotifyDaemon {
         this._init();
     }
 
+    set token(token) {
+        this.options.token = token;
+    }
+
+    reload() {
+        this.destroy();
+        this._killed = false;
```

Comment:
Is there a race here where the exit notification comes after we already set _killed = false and we restart twice?